src/test/java/pl/touk/sputnik/processor/detekt/DetektProcessorTest.java

```diff
@@ -87,6 +88,18 @@ public void shouldReturnNoViolationsForEmptyReview() {
         assertThat(result.getViolations()).isEmpty();
     }
 
+    @Test
```

Comment:
\[Checkstyle\] ERROR: Method 'shouldProcessReviewsOnDefaultConfig' is not designed for extension \- needs to be abstract, final or empty\.